suricata.yaml.in

```diff
@@ -238,6 +238,7 @@ outputs:
             #md5: [body, subject]
 
         #- dnp3
+        #- tftp
```

Comment:
should this be enabled by default if rust is built in? So using @rust_config_comment@

Reply:
Seems a app-layer config option is missing

Reply:
First comment is understood (I will add the @rust_config_comment@- tftp).
I do not understand your last comment. Thanks.